content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -625,174 +479,85 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
 You can also add a JWT policy to an ingress gateway (e.g., service `istio-ingressgateway.istio-system.svc.cluster.local`).
 This is often used to define a JWT policy for all services bound to the gateway, instead of for individual services.
 
-### End-user authentication with per-path requirements
+### Require a valid token
 
-End-user authentication can be enabled or disabled based on request path. This is useful if you want to
-disable authentication for some paths, for example, the path used for health check or status report.
-You can also specify different JWT requirements on different paths.
-
-{{< warning >}}
-The end-user authentication with per-path requirements is an experimental feature in Istio 1.1 and
-is **NOT** recommended for production use.
-{{< /warning >}}
-
-#### Disable End-user authentication for specific paths
-
-Modify the `jwt-example` policy to disable End-user authentication for path `/user-agent`:
+To reject requests without valid tokens, add an authorization rule specifying a `DENY` action for requests without request principals, shown as `notRequestPrincipals: ["*"]` in the following example. Request principals are available only when valid JWT tokens are provided. The rule therefore denies requests without valid tokens.
```

Comment:
in the following example, can be put in the end of paragraph. here is jump-in and then switch to explaining.

Reply:
You are correct, but I thought that was a minor issue. I'm fine if you move it to the end:
```suggestion
To reject requests without valid tokens, add an authorization rule specifying a `DENY` action for requests without request principals, shown as `notRequestPrincipals: ["*"]`. Request principals are available only when valid JWT tokens are provided. The rule therefore denies requests without valid tokens as seen in the following example:
```